assets/src/edit-story/app/media/utils/createResource.js

```diff
@@ -81,6 +82,7 @@ function createResource({
     title,
     alt,
     local,
+    sizes,
```

Comment:
Need to update the `Resource` type definition above to include this new property

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/837/commits/98b17a3d92282a84f9809fa8350f4c70f7319873